api/background/handlers/kaspa/connect.ts

```diff
@@ -42,7 +42,12 @@ export const connectHandler: Handler = async (
   }
 
   // Check if connection is already existing
-  if (await ApiUtils.isHostConnected(message.host)) {
+  if (
+    await ApiUtils.isHostConnectedWithNetworkId(
```

Comment:
This will prevent the `switch-network` request to succeed

Reply:
[2025-04-15 11-40-18.webm](https://github.com/user-attachments/assets/fd2b71bd-e9a8-4c8e-917c-2761ddc86544)

It works in the case. Am I missing other cases?

Reply:
Nevermind if it works let's go ahead